lightly_studio/tests/api/routes/api/test_embeddings2d.py

```diff
@@ -64,7 +64,8 @@ def test_get_embeddings2d__2d(
             dataset_id=dataset_id,
         ).samples
     ]
-    assert sample_ids == expected_sample_ids
+    assert len(sample_ids) == n_samples
+    assert set(sample_ids) == set(expected_sample_ids)
```

Comment:
This change was needed because of the change in embeddings resolver ordering. Feel free to keep it or remove it - I need to introduce it for my PR anyway.